### PR TITLE
million quasar catalog

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR continues the series requesting external catalogs (this time Vizier).

### CODE REVIEW

File: skyportal/handlers/api/annotation_services.py [hunk 1]
```diff
@@ -146,3 +147,133 @@ def post(self, obj_id):
...
+            get cross-match with Vizier and post them as an annotation
+            based on cross-matches to some catalog
+            (default is VII/290, i.e. the million quasar catalog).
+        parameters:
+          - in: path
+            name: obj_id
+            required: true
+            schema:
+              type: string
+            description: ID of the object to retrieve WISE colors for
```
Comment (guynir42) [inline skyportal/handlers/api/annotation_services.py:167]: this is no longer about WISE.

File: skyportal/handlers/api/annotation_services.py [hunk 2]
```diff
@@ -146,3 +147,133 @@ def post(self, obj_id):
...
+                data=annotation_data,
+                obj_id=obj_id,
+                origin=origin,
+                author=author,
+                groups=groups,
+            )
+            annotations.append(annotation)
+
+        if len(annotations) == 0:
+            return self.error("No QSO flag available.")
```
Comment (guynir42) [inline skyportal/handlers/api/annotation_services.py:267]: I'm not sure how QSO flag is related to this.

File: skyportal/tests/api/test_annotation_services.py [hunk 3]
```diff
@@ -51,3 +51,44 @@ def test_irsa_wise(public_group, upload_data_token):
             for d in data['data']
         ]
     )
+
+
+@pytest.mark.flaky(reruns=2)
+def test_vizier_quasar(
+    annotation_token, public_source, public_group, upload_data_token
```
Comment (guynir42) [inline skyportal/tests/api/test_annotation_services.py:58]: Don't need `public_source`.

Comment (guynir42) [reply skyportal/tests/api/test_annotation_services.py:58]: and also maybe not need `annotation_token`.

**Review by guynir42 — APPROVED** [review]
Minor comments.

File: skyportal/handlers/api/annotation_services.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/annotation_services.py]: yup.

File: skyportal/handlers/api/annotation_services.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/annotation_services.py]: me neither.

File: skyportal/tests/api/test_annotation_services.py (hunk 3 above)
Comment (mcoughlin) [reply skyportal/tests/api/test_annotation_services.py]: Indeed.

**Pull request merged**